<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableView" name="tableExperiments">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="bttnDeleteExperiment">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttnPlot">
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttnExcel">
           <property name="text">
            <string>Excel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Significant from:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSignificant"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Normalize to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboNormalize"/>
         </item>
         <item>
          <widget class="QPushButton" name="bttnCompare">
           <property name="text">
            <string>Compare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttnCopy">
           <property name="text">
            <string>Copy Rows</string>
           </property>
           <property name="shortcut">
            <string>Meta+C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttnPaste">
           <property name="text">
            <string>Paste Rows</string>
           </property>
           <property name="shortcut">
            <string>Meta+V</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tablePlates">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>800</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="bttnAddPlates">
           <property name="text">
            <string>Add Plates</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="bttnDeletePlate">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft">
     <widget class="QFrame" name="horizontalFrame_2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="horizontalFrame">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cell Line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCellLines">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Filter (comma separated):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtFilter">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
